docs/source/appendices/firewall-notes.md

```diff
@@ -0,0 +1,23 @@
+# Notes on Firewall Setup
+
+When you set up the firewall (or security group on AWS) for a BigchainD node, here is a list of the ports where _unsolicited_ inbound traffic is expected:
+
+22 - The default SSH port (TCP) <br>
+8080 - The default port for the RethinkDB web interface (TCP) <br>
```

Comment:
port `8080` and `28015` should probably be closed to the outside. The web ui should only be accessed by the admin of the node and the client driver port should only be accessed by the bigchaindb layer which is running in the same node